Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -170,6 +170,10 @@ class OWTestLearners(OWWidget):
     TARGET_AVERAGE = "(Average over classes)"
     class_selection = settings.ContextSetting(TARGET_AVERAGE)
 
+    shown_scores = \
+        settings.Setting({"AUC", "CA", "F1", "Precision", "Recall",
```

Comment:
Non-primitive settings cannot be stored to workflow in json format resulting in base64 encoded pickled blobs. Not sure if we care, since the widget uses context settings, which also cannot be serialized with json.